src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class Connectedk8sScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_connectedk8s')
+    def test_connectedk8s(self, resource_group):
+
+        self.kwargs.update({
+            'name': 'test1',
+            'resource_group': resource_group
```

Comment:
This can be removed. You have used the default key `rg` for the resource group created with the preparer.